src/studio/nuxeo-studio/working-in-studio/templates.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Templates
+review:
+    comment: ''
+    date: '2018-10-25'
+    status: ok
+tree_item_index: 650
+toc: true
+---
+## Document Templates
+
+Document templates are used as conversion instructions, to render XML, HTML, CSV or any open format from a document or a list of documents, using specific rendering operations. You would especially want to use this when you handle business data on content types and want to generate a nicer presentation for publishing, export, etc.
```

Comment:
I would add: "This is especially needed when you wish to create a file from document properties, and create file on demand".